cmd/manager/main.go

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Does it matter if this file was created entirely by the operator-sdk? If I go back and look at the old scripted generation logic, it seems like we delete/recreate this code every time without edits. I see @mangelajo made one commit to it after it was moved, but that was just to change these paths:

```
-       "github.com/submariner-operator/submariner-operator/pkg/apis"
-       "github.com/submariner-operator/submariner-operator/pkg/controller"
+       "github.com/submariner-io/submariner-operator/pkg/apis"
+       "github.com/submariner-io/submariner-operator/pkg/controller"
```

Non-blocker

Reply:
It should be fine, as far as I remember operator ask does not impose a license .

Reply:
But good point